walletlibrary/src/main/java/com/microsoft/walletlibrary/mappings/presentation/PresentationResponseMapping.kt

```diff
@@ -32,7 +32,9 @@ private fun PresentationResponse.addVerifiedIdRequirement(verifiedIdRequirement:
             VerifiedIdExceptions.REQUIREMENT_NOT_MET_EXCEPTION.value
         )
     val credentialPresentationInputDescriptor =
-        request.getPresentationDefinition().credentialPresentationInputDescriptors.filter { it.id == verifiedIdRequirement.id }
+        request.getPresentationDefinitions().filter { it.id == this.requestedVcPresentationDefinitionId }.map { it.credentialPresentationInputDescriptors
```

Comment:
Format this, so that it makes it a little bit easier to read / what does this do? Maybe add a comment?